dowhy/causal_refuter.py

```diff
@@ -108,11 +110,11 @@ def test_significance(self, estimate, simulations, test_type='auto',significance
             p_value = self.perform_normal_distribution_test(estimate, simulations)
 
         else:
-            raise Exception("{} has not been implemented".format(test_type))
+            raise NotImplementedError
 
         significance_dict = {
                 "p_value":p_value,
-                "pass_test": p_value <= significance_level
+                "fail_test": p_value > significance_level
```

Comment:
Shouldn't this be <= If the p-value is less than 0.05, then that means that the original estimate is unlikely given the null distribution, then that implies that the estimate is from the true value.

Reply:
I change the name to a little more neutral one.